MetadataExtractor/Formats/Png/PngMetadataReader.cs

```diff
@@ -207,9 +215,11 @@ private static IEnumerable<Directory> ProcessChunk([NotNull] PngChunk chunk)
             else if (chunkType == PngChunkType.tEXt)
             {
                 var reader = new SequentialByteArrayReader(bytes);
-                var keyword = reader.GetNullTerminatedString(maxLengthBytes: 79);
+                var keyword = reader.GetNullTerminatedStringValue(maxLengthBytes: 79).ToString(defaultEncoding);
                 var bytesLeft = bytes.Length - keyword.Length - 1;
```

Comment:
I think there's a bug here for encodings where `string.Length != bytes.Length` (multi-byte characters).